### PR TITLE
9418 student login view

### PR DESCRIPTION
**Pull request by hieptq-0855**
https://edu-redmine.sun-asterisk.vn/issues/9418

### CODE REVIEW

**Comment by hieptq-0855** [comment]
ready

**Comment by hieptq-0855** [comment]
ready

**Comment by hieptq-0855** [comment]
ready

File: public/css/login.css [hunk 1]
```diff
@@ -0,0 +1,18 @@
+div.register-background
+{
+	background-image: url(/student_img/hoc_sinh_khac_sinh_vien.jpg);
```
Comment (BunnyPi04) [inline public/css/login.css]: indent = 4 spaces

File: routes/web.php [hunk 2]
```diff
@@ -11,6 +11,12 @@
...
 
-Route::get('/', function () {
-    return view('welcome');
+Route::get('/', 'RouteController@returnWelcome');
+
+Route::get('student/login', 'RouteController@returnLogin');
+
+Route::get('student/register', 'RouteController@returnRegister');
+
+Route::group(['prefix'=>'student'], function(){
```
Comment (BunnyPi04) [inline routes/web.php]: convention

**Comment by hieptq-0855** [comment]
ready

**Comment by hieptq-0855** [comment]
ready

**Comment by hieptq-0855** [comment]
ready

File: resources/css/welcome.css [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    font-weight: 600;
+    letter-spacing: .1rem;
+    text-decoration: none;
+    text-transform: uppercase;
+}
+
+.m-b-md {
+    margin-bottom: 30px;
+}
+
```
Comment (chinn01) [inline resources/css/welcome.css]: dòng trắng cuối file

File: resources/css/login.css [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+
+div.register-background
+{
+    background-size: cover; 
+    background-position: center;
+    background-image: url(/student_img/hoc_sinh_khac_sinh_vien.jpg);
+}
+div.margin-left-15
+{
+	margin-left: 15px;
```
Comment (BunnyPi04) [inline resources/css/login.css]: indent = 4 spaces

File: resources/views/welcome.blade.php [hunk 5]
```diff
@@ -3,95 +3,22 @@
     <head>
         <meta charset="utf-8">
         <meta name="viewport" content="width=device-width, initial-scale=1">
-
-        <title>Laravel</title>
-
+        
+        <title>Trang quản lý sinh viên</title>
```
Comment (BunnyPi04) [inline resources/views/welcome.blade.php]: i18n

**Comment by hieptq-0855** [comment]
ready

**Comment by hieptq-0855** [comment]
ready

File: resources/lang/en/lang.php [hunk 6]
```diff
@@ -0,0 +1,7 @@
+<?php
+	return [
```
Comment (BunnyPi04) [inline resources/lang/en/lang.php]: indent = 4 spaces nhé

File: resources/lang/vi/lang.php [hunk 7]
```diff
@@ -0,0 +1,8 @@
+<?php
+	return [
+		'welcome_title' => 'Trang quản lý sinh viên',
+		'welcome_content' => 'Quản lý sinh viên',
+		'welcome_admin' => 'Trang giành cho admin',
+		'welcome_student' => 'Trang giành cho sinh viên',
+	];
+
```
Comment (BunnyPi04) [inline resources/lang/vi/lang.php]: thiếu dòng trắng cuối file

File: resources/lang/vi/lang.php [hunk 8]
```diff
@@ -0,0 +1,8 @@
+<?php
+	return [
+		'welcome_title' => 'Trang quản lý sinh viên',
+		'welcome_content' => 'Quản lý sinh viên',
+		'welcome_admin' => 'Trang giành cho admin',
```
Comment (BunnyPi04) [inline resources/lang/vi/lang.php]: `Dành` not `giành`

**Comment by hieptq-0855** [comment]
ready

**Pull request merged**